README.md

```diff
@@ -0,0 +1,42 @@
+# Paste HTML to govspeak
+
+Converts HTML formatted rich content to [govspeak][] format (a markdown extension library for government editors) when pasted from clipboard into a form input or textarea.
+
+## Installation
+
+Add the compiled `/dist/paste-html-to-markdown.js` to your application assets.
+
+An npm installation will be available at a future point.
+
+## Usage
+
+```js
+element.addEventListener('paste', window.pasteHtmlToGovspeak)
+```
+
+## Browser support
+
+- Chrome
+- Firefox
+- Safari
+- Internet Explorer 11
+- Microsoft Edge
+
+[govspeak]: https://github.com/alphagov/govspeak
```

Comment:
Sorry, minor but this link seems to have ended up in the middle of this file.

Reply:
Good spot. Moved at the end of the file.